docs/pages/KCheckbox.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+
+  <DocsPageTemplate apiDocs>
+
+    <DocsPageSection title="Overview" anchor="#overview">
+      <p>These are checkbox components:</p>
+      <DocsShow>
+        <KCheckbox
+          label="Some label"
+          :checked="true"
+        />
+        <KCheckbox
+          label="Another label"
+          :checked="false"
+        />
+      </DocsShow>
+      <p>
+        The checkbox is generally used to select one of two possible values in a form. It should not be used to make real-time changes; for this situation, use a <code>KSwitch</code> component instead.
+      </p>
+      <p>Layout:</p>
+      <ul>
+        <li>Always left-aligned</li>
```

Comment:
This is a bit confusing. In RTL this would be right-aligned. If you say "always" I'd think that this is true even when in RTL.

Is there a proper term for "text-direction aligned"?

Reply:
updated